telcell/data/parsers.py

```diff
@@ -42,7 +48,10 @@ def parse_measurements_csv(path: Union[str, Path]) -> List[Track]:
                 Measurement(
                     lat=float(row['celldb.wgs84.lat']),
                     lon=float(row['celldb.wgs84.lon']),
-                    timestamp=row['timestamp'],
+                    timestamp=datetime.datetime.strptime(
+                        row['timestamp'],
+                        "%Y/%m/%d, %H:%M:%S"
```

Comment:
Mijn tijdzone-alarmbellen gaan af, weten we zeker dat we voor een vers stuk code willen werken met tz-loze datetime's? :(

Reply:
Nee dat willen we zeker niet, ik had deze parser aangepast zodat hij zou werken met de test `.csv` bestanden die we hebben en die bevatten geen informatie over timezone. Vandaar dat ik daar hier ook niets mee kon.

Ik begrijp van @wowtor inmiddels dat in onze "echte" `measurements.csv` de timestamps anders geformatteerd zijn en die informatie wel aanwezig is, dus ik ga zowel deze code als de test data aanpassen naar dat nieuwe formaat waarbij de timezone dus wel mee wordt genomen.